api/api.go

```diff
@@ -13,29 +14,43 @@ import (
 	"github.com/jinzhu/gorm"
 	"github.com/netlify/netlify-commerce/conf"
 	"github.com/netlify/netlify-commerce/mailer"
+	"github.com/pborman/uuid"
 	"github.com/rs/cors"
-	"github.com/zenazn/goji/web/mutil"
 )
 
 var bearerRegexp = regexp.MustCompile(`^(?:B|b)earer (\S+$)`)
 
 // API is the main REST API
 type API struct {
 	handler    http.Handler
+	listener   net.Listener
```

Comment:
Use the httptest framework to test http end to end if we need to https://golang.org/pkg/net/http/httptest